src/components/Widgets/ListControl.css

```diff
@@ -47,6 +47,18 @@
   border: none;
   border-radius: var(--borderRadius) 0 0 var(--borderRadius);
   background: rgba(0,0,0,0.1);
+
+  &:before {
```

Comment:
Feels like this shouldn't be necessary - can't we just set the background color of the individual list items?

Reply:
@erquhart this is for the tab with the arrow icon, it remains semi-transparent otherwise. But I can remove if it is not necessary.